homeassistant/components/sensor/__init__.py

```diff
@@ -74,3 +75,7 @@ async def async_setup_entry(hass, entry):
 async def async_unload_entry(hass, entry):
     """Unload a config entry."""
     return await hass.data[DOMAIN].async_unload_entry(entry)
+
+
+class SensorEntity(Entity):
```

Comment:
Please rebase on latest dev branch.

Reply:
This will disappear when squashing+merging. Is it confusing when reviewing?

Reply:
Yes, we shouldn't have changes in PRs where the changes are already merged in the target branch.

Reply:
Alright, fixed.